spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubOperations.java

```diff
@@ -16,14 +16,39 @@
 
 package org.springframework.cloud.gcp.pubsub.core;
 
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.cloud.pubsub.v1.Subscriber;
+
 import org.springframework.messaging.Message;
 import org.springframework.util.concurrent.ListenableFuture;
 
 /**
+ * An abstraction for Google Cloud Pub/Sub.
+ *
  * @author Vinicius Carvalho
+ * @author João André Martins
  */
 public interface PubSubOperations {
 
-	ListenableFuture<String> send(String topic, Message message);
+	/**
+	 * Sends a Spring {@link Message} to Pub/Sub.
+	 *
+	 * @param topic the name of an existing topic
+	 * @param message the Spring message
+	 * @return the listenable future of the call
+	 */
+	ListenableFuture<String> publish(String topic, Message message);
```

Comment:
As I commented before, I think this method should take `PubsubMessage` and keep the messaging APIs out of `PubSubOperations`/`PubSubTemplate`. This would make it consistent with `JmsTemplate` and `RabbitTemplate`. If we think that a `PubSubMessagingTemplate` is useful for channel adapters, we can create that as well.

Reply:
This PR concerns only adding subscribe() to `PubSubOperations`.
If you don't mind, I'd prefer filing a new issue to add `PubsubMessage` payload support, or even address it in an open PR like #151, although I think we need to discuss before, as I'm not sure if surfacing `PubsubMessage` is useful, or a good idea.